### ISSUE TITLE
[OneNote] Importer does not respect the "Use [[Wikilinks]]" setting

### ISSUE THREAD

**Issue by FeralFlora** [obsidianmd/obsidian-importer#265]
![image](https://github.com/obsidianmd/obsidian-importer/assets/33608083/e9db2144-a222-456b-892f-200e33487b28)

Even though I have the "Use [[Wikilinks]]" toggle enabled, all my image embeds are in markdown link format. This seems to be due to the use of `htmlToMarkdown`:

https://github.com/obsidianmd/obsidian-importer/blob/4be63256cae2c5d852ee1ed75f50d81b87f801c2/src/formats/onenote.ts#L321

https://github.com/obsidianmd/obsidian-importer/blob/4be63256cae2c5d852ee1ed75f50d81b87f801c2/src/formats/html.ts#L221

The file link is also hardcoded to be URI encoded, which is not necessary in wikilinks:
https://github.com/obsidianmd/obsidian-importer/blob/4be63256cae2c5d852ee1ed75f50d81b87f801c2/src/formats/onenote.ts#L543

**Comment by tgrosinger** [comment]
Yes, this is behavior from htmlToMarkdown as you mentioned. It's not something that can be changed by this plugin, so I'm going to close this issue for now.
